docs/tool/js/views/map-view.js

```diff
@@ -869,6 +871,59 @@
                 .remove();
 
         },
+        exportCsv: function() {
+          // if (data){
+            d3.select('#csvExport')
+              .on('click', function(d) {
+                  console.log("YOU CLICKED EXPORT CSV");
+                  var preview = d3.select('#buildings-list');
+                  var listItems = preview.selectAll('div');
+                  var projectNames = [];
+                  listItems._groups[0].forEach( function(project){
+                    projectNames.push(project.innerText);
+                  })
+                  var allData = mapView.convertedProjects.features;
```

Comment:
Instead of using the geoJson, can you get the data  from `model.dataCollection['raw_project']`? We are planning to remove some of the duplicative data from the geojson; also, this doesn't have the 'klass' column that is only relevant to mapbox.

Reply:
Done